pkg/virt-launcher/monitor.go

```diff
@@ -156,41 +186,65 @@ func readProcCmdline(pathname string) ([]string, error) {
 	return strings.Split(string(content), "\x00"), nil
 }
 
-func pidOf(exename string) (int, error) {
+func pidExistsInMyCgroup(pid int) (bool, error) {
```

Comment:
I think we can now change detection to first look up all processes in the cgroup, and and then checking the  returned processes if one of them is a qemu process.